### PR TITLE
Enabled ssl connections to redis

### PR DESCRIPTION
**Pull request by smurariu2**
This is needed when monitoring azure redis instances.

### CODE REVIEW

File: Opserver.Core/Settings/RedisSettings.cs [hunk 1]
```diff
@@ -50,9 +50,15 @@ public class Instance : ISettingsCollectionItem
             public string Password { get; set; }
 
             /// <summary>
+            /// Specify if ssl should be used. Defaults to false.
+            /// </summary>
+            public bool Ssl { get; set; } = false;
```
Comment (NickCraver) [inline Opserver.Core/Settings/RedisSettings.cs]: let's call this `UseSSL` so it's clear that it's a bool and consistent with naming (e.g. "CPU" is all caps in settings)

**Comment by NickCraver** [comment]
One renaming for consistency needed - happy to merge in after that!

File: Opserver.Core/Settings/RedisSettings.cs (hunk 1 above)
Comment (smurariu2) [inline Opserver.Core/Settings/RedisSettings.cs]: You'll laugh but it was called that initially. Renamed just before the pull request.

**Comment by smurariu2** [comment]
Done

**Comment by NickCraver** [comment]
Thanks!

**Pull request merged**